languages/csharp/reference/exercise-concepts/grade-school.md

```diff
@@ -0,0 +1,25 @@
+# Concepts of grade-school
+
+[Example implementation](https://exercism.io/tracks/csharp/exercises/grade-school/solutions/2fd52e1bbfcecdfaff4075c4)
```

Comment:
Love your solution... it should definitely replace the current example that we have :)
https://github.com/exercism/csharp/blob/master/exercises/grade-school/Example.cs

Reply:
Agreed :)